src/main/java/frc/robot/subsystems/drivetrain/commands/SwitchGears.java

```diff
@@ -0,0 +1,22 @@
+package frc.robot.subsystems.drivetrain.commands;
+
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import frc.robot.Constants;
+import frc.robot.subsystems.drivetrain.Drivetrain;
+
+public class SwitchGears extends InstantCommand {
+    private Drivetrain drivetrain;
+
+    public SwitchGears(Drivetrain drivetrain) {
+        this.drivetrain = drivetrain;
+        addRequirements(drivetrain);
+    }
+
+    @Override
+    public void initialize() {
+        if (drivetrain.getSpeedLeft() > Constants.Drivetrain.MINIMUM_SPEED && drivetrain.getSpeedLeft() > Constants.Drivetrain.MINIMUM_SPEED)
```

Comment:
The safety feature should be in the subsystem and also include a cooldowm between each shift

Reply:
There should also be a safety feature that makes sure the robot is not turning